custom/icds_reports/utils/aggregation_helpers/distributed/ccs_record_monthly.py

```diff
@@ -31,17 +32,22 @@ def aggregate(self, cursor):
         for query in index_queries:
             cursor.execute(query)
 
+    def get_table(self, table_id):
+        if self.month != transform_day_to_month(date.today()):
+            return get_prev_table(table_id)
+        return get_table_name(self.domain, table_id)
```

Comment:
now seeing this repeated makes me think it should be a well-named helper function